CHANGELOG.md

```diff
@@ -5,10 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
-
 ## [Unreleased]
 
+## [1.0.0] 2020-05-19
+
+## Added
 
+- First release.
 
-[Unreleased]: https://github.com/giantswarm/etcd-backup-operator/tree/master
+[Unreleased]: https://github.com/giantswarm/etcd-backup-operator/compare/v1.0.0...HEAD
+[1.0.0]: https://github.com/giantswarm/etcd-backup-operator/releases/tag/v1.0.0
```

Comment:
Maybe not so smart question, is this intended to be hidden at the page?

Asking because it is not visible:

![image](https://user-images.githubusercontent.com/5638639/82464764-bdfa9080-9abe-11ea-84da-8653b781b5d2.png)

Reply:
Figured out, thanks to Jose! :)